src/WebJobs.Extensions.OpenAI.CosmosDBSearch/WebJobs.Extensions.OpenAI.CosmosDBSearch.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Microsoft Azure WebJobs SDK Extension for OpenAI - Cosmos DB Search for MongoDB Package.</Description>
+    <Version>$(CosmosDBSearchVersion)</Version>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Search.Documents" Version="11.5.1" />
```

Comment:
Does the Cosmos DB implementation also require an Azure Search SDK dependency? I didn't see this namespace mentioned in the `using` statements at the top of these class files.

Reply:
nope removed it!